x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -6,22 +6,21 @@
  */
 
 import React from 'react';
+import { useAssetDetailsStateContext } from '../hooks/use_asset_details_state';
 import { useTabSwitcherContext } from '../hooks/use_tab_switcher';
-import { Anomalies, Metadata, Processes, Osquery, Metrics, Logs, Overview } from '../tabs';
-import { FlyoutTabIds, type TabState, type AssetDetailsProps } from '../types';
+import { Anomalies, Metadata, Processes, Osquery, Logs, Overview } from '../tabs';
+import { FlyoutTabIds, type TabState } from '../types';
+import { toTimestampRange } from '../utils';
 
-type Props = Pick<
-  AssetDetailsProps,
-  'currentTimeRange' | 'node' | 'nodeType' | 'overrides' | 'onTabsStateChange'
->;
+export const Content = () => {
+  const {
+    node,
+    nodeType,
+    overrides,
+    dateRange: timeRange,
```

Comment:
Q: Why is this named `timeRange` here ( I guess now this is the one coming from the searchCriteria that was in the overrides, right)? IMO it's getting a bit confusing with all time/date range names ( the Timestamp naming is good)

Reply:
I forgot to rename this. It should be `dateRange`